.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+-   [ ] Bug fix (non-breaking change which fixes an issue)
+-   [ ] New feature (non-breaking change which adds functionality)
+-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+-   [ ] Dependency update (can be both non-breaking and breaking change)
+-   [ ] This change requires a documentation update
+
+# Checklist:
```

Comment:
too much extra work to tick these (and they should be followed by default 😅 ) but we can keep the `type of change`